packages/astro/package.json

```diff
@@ -168,6 +168,7 @@
     "semver": "^7.5.4",
     "server-destroy": "^1.0.1",
     "shikiji": "^0.9.18",
+    "shikiji-core": "^0.9.18",
```

Comment:
You didn't update the version at all; it's the same as before: `0.9.18`

Reply:
I don't think the version needs to change. It's only moving it from devdeps to deps.

Reply:
Do you want to update the shikiji-core ?

Reply:
> I don't think the version needs to change. It's only moving it from devdeps to deps.

Ah. The PR doesn't mention that. I wished there was more context in the description.